src/Features/RemoteFreeExtensions/Init.php

```diff
@@ -21,6 +21,7 @@ class Init {
 	 */
 	public function __construct() {
 		add_action( 'change_locale', array( __CLASS__, 'delete_specs_transient' ) );
+		add_action( 'woocommerce_admin_updated', array( __CLASS__, 'delete_specs_transient' ) );
```

Comment:
@joshuatf I believe this constructor is never called, as all the other functions are static, so it doesn't get instantiated anywhere? Or at-least my break points never hit this, or an error log I added.

Reply:
@louwie17 Good call.  I think the feature name was incorrect, causing this constructor to never get called.  

Updated this; note that `npm run build:feature-config` will need to be run to rebuild the feature config file.

Reply:
Quick note: I also didn't update this feature name in the business details step given it's being removed from there in https://github.com/woocommerce/woocommerce-admin/pull/7452.

Reply:
Thanks, that indeed fixed it :)